models/user.model.js

```diff
@@ -21,9 +21,19 @@ exports._get = (id) => {
 
 exports._post = (user,ctx) => {
   return new Promise ((resolve, reject) => {
-    conn.query('INSERT INTO users SET ?', user , (err, res) => {
+    conn.query('INSERT INTO users SET ?', [user] , (err, res) => {
       if (err) return resolve(err);
-      resolve(res);
+      resolve(res)
+    })
+  })
+};
+
+exports._postResponse = (id,ctx) => {
+  return new Promise ((resolve, reject) => {
+
+    conn.query('SELECT * FROM users WHERE user_id = ?', [id], (err, res) => {
```

Comment:
This is a `getUser`. 

Why are you prepending all function with underscore?

Reply:
I use it only for the functions which are a HTTP request as convention. is it a bad practice ?